pom.xml

```diff
@@ -820,5 +898,62 @@
         <maven.skip.it.tests>true</maven.skip.it.tests>
       </properties>
     </profile>
+    <profile>
+      <!-- Need to load the appropriate driver for the particular DB to test based on a system path -->
+      <id>database</id>
+      <activation>
+        <property>
+          <name>maven.jdbc.db-kind</name>
```

Comment:
if that profile gets activated, yes, the driver path should exist, otherwise it will complain - but, this profile doesn't get activated. Running `mvn help:active-profiles` displays:

`
The following profiles are active:

 - custom-container (source: org.kie:process-migration-service:7.70.0-SNAPSHOT)
 - skipITs-by-default (source: org.kie:process-migration-service:7.70.0-SNAPSHOT)
 - doclint-java8-disable (source: org.jboss:jboss-parent:36)
`